kratos/python/add_accessors_to_python.cpp

```diff
@@ -20,9 +20,11 @@
 #include "add_accessors_to_python.h"
 #include "includes/define_python.h"
 #include "includes/accessor.h"
+#include "includes/table_accessor.h"
 #include "includes/properties.h"
 
-using AccessorBindType = std::unique_ptr<Kratos::Accessor>;
+using AccessorBindType      = std::unique_ptr<Kratos::Accessor>;
+// using TableAccessorBindType = std::unique_ptr<Kratos::TableAccessor>;
```

Comment:
exporting it to be decided

Reply:
I would not export and leave just the base Accessor visible to python, unless we have a very good reason to do otherwise, because the only point when the user would be aware of the specific class is between the creation and the assignment to a property object (and is probably better to just create the object directly in the assignement as @pooyan-dadvand suggested). After that once you `GetAccessor` the type is lost.

Reply:
I agree, THX